### PR TITLE
Update .prettierrc.json

### PR DESCRIPTION
**Pull request by camilovegag**
Adding some of the `.prettierrc.json` options that we use at hoprnet-org.

Of course we can check if those options are ok for us

https://prettier.io/docs/en/options.html

### CODE REVIEW

**Review by ItsLuized — APPROVED** [review]

**Comment by mjadach-iv** [comment]
Lets wait with merge until older PRs are merged.

**Review by mjadach-iv — CHANGES_REQUESTED** [review]
Rebase needed

File: .prettierrc.json [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
-    "trailingComma": "es5",
-    "tabWidth": 2,
-    "semi": true,
-    "singleQuote": true
-}
\ No newline at end of file
+  "printWidth": 80,
```
Comment (mjadach-iv) [inline .prettierrc.json:2]: Lets make it 120

Comment (camilovegag) [reply .prettierrc.json]: Updated

File: src/future-hopr-lib-components/NFT/nftImageInteractive.jsx [hunk 2]
```diff
@@ -167,18 +167,9 @@ export default function NftImageInteractive(props) {
...
-            rotated ? 'rotate' : '',
-          ].join(' ')}
-        >
+        <div className={['nft-front', fliped ? 'flip' : '', rotated ? 'rotate' : ''].join(' ')}>
           <div
-            className={[
-              'nft-image-container',
-              props.ignored ? 'ignored' : '',
-            ].join(' ')}
+            className={['nft-image-container', props.ignored ? 'ignored' : ''].join(' ')}
```
Comment (mjadach-iv) [inline src/future-hopr-lib-components/NFT/nftImageInteractive.jsx:172]: @camilovegag, please check if we can make an exception for arrays in props to have singleAttributePerLine

File: src/App.tsx [hunk 3]
```diff
@@ -1,11 +1,5 @@
 import { Store } from './types/index';
-import {
-  RouterProvider,
-  BrowserRouter,
-  Route,
-  createRoutesFromElements,
-  createBrowserRouter,
-} from 'react-router-dom';
+import { RouterProvider, BrowserRouter, Route, createRoutesFromElements, createBrowserRouter } from 'react-router-dom';
```
Comment (mjadach-iv) [inline src/App.tsx:2]: @camilovegag, please check if we can make an exception for imports to have singleAttributePerLine

File: src/future-hopr-lib-components/Layout/index.tsx [hunk 4]
```diff
@@ -37,14 +37,7 @@ const Layout: React.FC<{
...
   drawerItems?: {}[];
-}> = ({
-  className = '',
-  children,
-  itemsNavbarRight,
-  tallerNavBarOnMobile,
-  drawer,
-  drawerItems,
-}) => {
+}> = ({ className = '', children, itemsNavbarRight, tallerNavBarOnMobile, drawer, drawerItems }) => {
```
Comment (mjadach-iv) [inline src/future-hopr-lib-components/Layout/index.tsx:40]: @camilovegag, please check if we can make an exception for destructuring like this one to have singleAttributePerLine

**Comment by mjadach-iv** [comment]
The branch is rebased onto current develop

**Comment by camilovegag** [comment]
Closing as this will be addressed on #56

**Pull request closed without merging**